src/lib/hooks/useDraggableScrollView.ts

```diff
@@ -51,7 +51,7 @@ export function useDraggableScroll<Scrollable extends ScrollView = ScrollView>({
         return
       }
 
-      // Require n pixels momement before start of drag (3 in this case )
+      // Require n pixels moment before start of drag (3 in this case )
```

Comment:
```suggestion
      // Require n pixels movement before start of drag (3 in this case)
```
Given the context, I think it probably should be `movement` rather than `moment`?